platform/osx/os_osx.mm

```diff
@@ -1494,9 +1494,19 @@ static void keyboardLayoutChanged(CFNotificationCenterRef center, void *observer
 
 };
 
+const int menuHeight = [[NSStatusBar systemStatusBar] thickness];
```

Comment:
Is that really a const? What would happen if the user changes the height of the systemStatusBar while Godot is running? :D